OpenRA.Mods.Common/Activities/Enter.cs

```diff
@@ -145,7 +146,10 @@ ReserveStatus TryReserveElseTryAlternateReserve(Actor self)
 						Target t = target;
 						if (!TryGetAlternateTarget(self, tries, ref t))
 							return ReserveStatus.TooFar;
-						if ((target.CenterPosition - self.CenterPosition).HorizontalLengthSquared <= (t.CenterPosition - self.CenterPosition).HorizontalLengthSquared)
+
+						var targetPosition = target.Positions.PositionClosestTo(self.CenterPosition);
+						var alternatePosition = t.Positions.PositionClosestTo(self.CenterPosition);
```

Comment:
Shouldn't we use here bool targetCenter to decide the target pos?
or in this code 
`inner = move.MoveToTarget(self, targetCenter ? Target.FromPos(target.CenterPosition) : target); // Approach`
also use PositionClosestTo? Though for ReserveStatus.TooFar it probably doesn't matter too much.